RipCurrent/test.cc

```diff
@@ -978,3 +982,56 @@ TEST(RipCurrent, SchedulingEventsIntoTheFutureAndUpdatingHead) {
   }
   EXPECT_EQ("11,12,17,19", current::strings::Join(result, ','));
 }
+
+namespace ripcurrent_unittest {
+
+// clang-format off
+RIPCURRENT_NODE_T(TemplatedEmitter, void, T) {
+  TemplatedEmitter() {
+    // Too bad we need this ugly syntax for now. -- D.K.
+    RIPCURRENT_TEMPLATED_EMIT((T), T, LifeUniverseAndEverything());
```

Comment:
Yes, and to confirm the macro does "run" `REMOVE_PARENTHESES`. I did build it w/o parens too.

This will soon be revisited regardless, as, in the presence of MMPQ, RipCurrent templating has  been the last piece we needed to begin working on a simple yet powerful "Standard RipCurrent library".